### ISSUE TITLE
SqueezeNet training on cifar

### ISSUE THREAD

**Issue by GitZinc** [DeepScale/SqueezeNet#54]
The accuracy is 0, and the loss is too high all the time when I run the model on cifar10.
Do I need to delete avg pooling layer?

**Comment by morawi** [comment]
Well, now I am getting the SqueezeNet to move (I am using PyTorch):
The problem was using an ImageNet pre-trained model, so setting pre-trained-model to False solved the problem.

**Comment by GitZinc** [comment]
Wow, I used caffe at that time. Thanks for your reply, now I think I find the problem.
